connector-drawer/src/main/resources/application.properties

```diff
@@ -34,5 +35,11 @@ camel.context.name=notifications-connector-drawer
 
 mp.messaging.tocamel.topic=platform.notifications.tocamel
 mp.messaging.fromcamel.topic=platform.notifications.fromcamel
+mp.messaging.todrawer.topic=platform.chrome
 
 %test.quarkus.log.category."com.redhat.cloud.notifications".level=DEBUG
+
+# The URL for the recipients-resolver.
+notifications.connector.recipients-resolver.url=${clowder.endpoints.notifications-recipients-resolver-service.url:http://localhost:9008}
+
+quarkus.http.test-port=9008
```

Comment:
Does this need to be included in the PR?

Reply:
Right, we don't